CMakeLists.txt

```diff
@@ -101,12 +103,17 @@ install(FILES snappy.h
         ${Snappy_BINARY_DIR}/snappy-stubs-public.h
         DESTINATION include)
 
+if (BUILD_SHARED_LIBS)
```

Comment:
I think most CMake projects end up adding a `_static` suffix to the library name. Honestly this is actually pretty nice as one can clearly specify they want static linking on Unix by simply using this other library name.